JHUGenerator/main.F90

```diff
@@ -580,7 +584,7 @@ SUBROUTINE GetCommandlineArgs()
     if( Process.eq.50 ) then
         DecayMode2=DecayMode1
         if( Collider.eq.2 ) call Error("Collider 2 not available for VH")
-        if( (IsAZDecay(DecayMode1).eqv..false.) .and. (Collider.ne.1) ) call Error("WH with Collider 1 only")
+        if( (IsAWDecay(DecayMode1) ) .and. (Collider.ne.1) ) call Error("WH with Collider 1 only")
```

Comment:
Ulascan please make sure this part is what you had in mind, I see you implemented e+e- --> gammaH so we might as well enable it